source/boundary_traction/function.cc

```diff
@@ -44,39 +44,9 @@ namespace aspect
                        const Tensor<1,dim> &) const
     {
       Tensor<1,dim> traction;
-      if (coordinate_system == Utilities::Coordinates::cartesian)
-        {
-          for (unsigned int d=0; d<dim; ++d)
-            traction[d] = boundary_traction_function.value(p,d);
-
-        }
-      else if (coordinate_system == Utilities::Coordinates::spherical)
-        {
-          const std_cxx11::array<double,dim> spherical_coordinates =
-            aspect::Utilities::Coordinates::cartesian_to_spherical_coordinates(p);
-          Point<dim> point;
-
-          for (unsigned int d=0; d<dim; ++d)
-            point[d] = spherical_coordinates[d];
-
-          for (unsigned int d=0; d<dim; ++d)
-            traction[d] = boundary_traction_function.value(point,d);
-
-        }
-      else if (coordinate_system == Utilities::Coordinates::depth)
-        {
-          const double depth = this->get_geometry_model().depth(p);
-          Point<dim> point;
-          point(0) = depth;
-
-          for (unsigned int d=0; d<dim; ++d)
-            traction[d] = boundary_traction_function.value(point,d);
-        }
-      else
-        {
-          AssertThrow(false, ExcNotImplemented());
-          return numbers::signaling_nan<Tensor<1,dim> >();
-        }
+      const Point<dim> point = this->get_geometry_model().cartesian_to_other_coordinates(p, coordinate_system);
```

Comment:
could you change the `p` here (and at the top of the function) to `point` to be more consistent with the rest of the code and because it is easier to search in the code for this variable.